V2 docs (#692)
bbcmicrobit/micropython: docs/music.rst

.. py:module:: music

This is the ``music`` module and you can use it to create and play melodies.

> By default sound output will be via the built-in speaker **V2** and via the 

Reviewer: As an earlier comment, maybe we should mention pin0 first as that's the common mode, and then indicate in v2 is also plays via speaker.

Author: Done in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5

edge connector on pin 0. You can connect a wired headphones or a speaker to
pin 0 and GND on the edge connector to hear the sound:

.. image:: music-pins.png

> You can change the default pin configuration to disable the built-in speaker

Reviewer: Technically that wouldn't disable the speaker, just not output the sound to it. Disabling it goes via pin_speaker.disable().

Author: Updated in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5

    Gets the current tempo as a tuple of integers: ``(ticks, bpm)``.

> .. py:function:: play(music, pin=microbit.pin_speaker, wait=True, loop=False)

Author: done

    their defaults before the music (whatever it may be) is played.

    An optional argument to specify the output pin can be used to override the
    default of ``microbit.pin0``. If you have the latest micro:bit, you can
    use ``microbit.pin_speaker`` **V2**.

    .. note::
        Using this argument will disable the default functionality on the
        **V2** board where the sound is mirrored on speaker and pin0, so
        ``pin=microbit.pin0`` will only output on the edge connector and

>         ``pin=microbit.pin_speaker`` will only use the built-in speaker.

Author: removed this note

    their defaults before the music (whatever it may be) is played.

    An optional argument to specify the output pin can be used to override the

>     default of ``microbit.pin0``. If you have the latest micro:bit, you can
>     use ``microbit.pin_speaker`` **V2**.
> 
>     The pin argument can also take a tuple of two pins, for example

Reviewer: Joining these two sentences, otherwise it sounds like you can use a tuple in V1 as well.

```suggestion
    default of ``microbit.pin0``. With micro:bit **V2** you can use
    ``microbit.pin_speaker`` or a tuple of two pins, for example
```
